src/main/java/hudson/plugins/jacoco/model/CoverageObject.java

```diff
@@ -203,7 +204,7 @@ public Coverage getLineCoverage() {
 	/**
 	 * Gets the build object that owns the whole coverage report tree.
 	 */
-	public abstract AbstractBuild<?,?> getBuild();
+	public abstract Run<?,?> getBuild();
```

Comment:
But again with Workflow we need this, so you are suggesting https://github.com/jenkinsci/bytecode-compatibility-transformer#adapting-type